### PR TITLE
Work-product submission

### PR DESCRIPTION
**Pull request by Saumya-Mishra9129**
@quozl @chimosky @pro-panda Please review and suggest if some changes are required.

### CODE REVIEW

**Comment by Saumya-Mishra9129** [comment]
Preview can be seen with  https://github.com/sugarlabs/GSoC/blob/0eefbbcd9eed23039f775eac5733bbb85eece880/archives/2020/student-reports/port-Sugar-and-core-activities-to-Python-3-Saumya-Mishra.md

**Review by JuiP — COMMENTED** [review]
Amazing Work @Saumya-Mishra9129 ! :) 

Review / suggestions : 
- Your product submission should have link to *merged* commits; not pull requests. (As mentioned in the work-product submission guidelines sent by gsoc)
- Clearly mention the work completed and the work that is left to complete. This way it is easier for other members / new contributors to continue your work.
- You may also mention your observations while debugging / a fix. This might be useful for new contributors.

**Comment by Saumya-Mishra9129** [comment]
> Amazing Work @Saumya-Mishra9129 ! :)
> 
> Review / suggestions :
> 
> * Your product submission should have link to _merged_ commits; not pull requests. (As mentioned in the work-product submission guidelines sent by gsoc)
> * Clearly mention the work completed and the work that is left to complete. This way it is easier for other members / new contributors to continue your work.
> * You may also mention your observations while debugging / a fix. This might be useful for new contributors.

Thanks :smiley:

**Comment by chimosky** [comment]
Reviewed, would also be nice to tabulate the work done as it'll be easier to go through.

**Comment by shaansubbaiah** [comment]
Cool stuff!😀 Might be useful to link your GitHub as well.

**Comment by Saumya-Mishra9129** [comment]
@quozl @pro-panda Please review and merge.

File: archives/2020/student-reports/improve-and-maintain-25-activities-Jui-Pradhan.md [hunk 1]
```diff
@@ -2,55 +2,67 @@ Improve and maintain atleast 25 Sugar Activities
 --------------------------
 ***Google Summer of Code, 2020***
 
-My work includes feature additions, bug fixes, adding support for different display resolutions, few design changes, port to python3 as a part of improving Sugar Activities. This project also aimed at maintaining activities, the work for which included: making activity releases, adding or fetching translations, flake8 changes and testing activities extensively. Moreover, fixing issues, reviewing pull requests and finishing unmerged work was also a part of maintenance.
+*Name - Jui Pradhan*
```
Comment (pro-panda) [inline archives/2020/student-reports/improve-and-maintain-25-activities-Jui-Pradhan.md:5]: You have modified @JuiP's work product. Was this intended?

Comment (Saumya-Mishra9129) [reply archives/2020/student-reports/improve-and-maintain-25-activities-Jui-Pradhan.md:5]: No Actually I just rebase my master branch with current master.

Comment (Saumya-Mishra9129) [reply archives/2020/student-reports/improve-and-maintain-25-activities-Jui-Pradhan.md]: @pro-panda Removed those commits. @quozl @chimosky Please review and merge as I need link to submit in evaluation.

**Comment by chimosky** [comment]
Reviewed, I'll wait for your mentors to merge when they're satisfied.

**Review by srevinsaju — APPROVED** [review]
Nice! :tada: 
Great to see that you mastered `gdb`. 
~~gdb is and remains a complex software for me to get started 🌟~~

**Comment by Saumya-Mishra9129** [comment]
> Reviewed, I'll wait for your mentors to merge when they're satisfied.
> If they don't, I'll merge later during the day as I understand today's the deadline to submit your final evaluations.

@chimosky ping

**Comment by chimosky** [comment]
@Saumya-Mishra9129 I apologize for the delay.

**Pull request merged**